### DISCUSSION TITLE
Support for key rotation

### DISCUSSION THREAD

**Discussion by nexxai** [spatie/laravel-ciphersweet#6] [Ideas]
This looks amazing, but without support for the source package's key rotation system (https://ciphersweet.paragonie.com/php/key-rotation), it feels a bit incomplete. Normally I wouldn't care if every feature is included in a wrapper but this option specifically would be extremely helpful in the case of a key compromise, when tensions are already high. Knowing that there would be a command I can run to cycle the encryption key would make me a lot more comfortable. Is there any chance that support for this will be added to the package?

**Comment by riasvdv** [comment]
The Encrypt command will actually rotate keys if you give it a new key (it will take your old key from the .env) https://github.com/spatie/laravel-ciphersweet/blob/main/src/Commands/EncryptCommand.php

**Reply by nexxai** [reply]
Even better! Thanks rias!
